src/routes/AgreementViewRoute.js

```diff
@@ -29,28 +28,33 @@ class AgreementViewRoute extends React.Component {
     agreementLines: {
       type: 'okapi',
       path: 'erm/entitlements',
+      limitParam: 'perPage',
+      perRequest: (_q, _p, _r, _l, props) => parseMclSettings(props.resources.settings, 'pageSize', 'agreementLines'),
```

Comment:
Needs to be tweaked according to changes made above with the parseMclSettings function